CONTRIBUTING.md

```diff
@@ -117,7 +118,45 @@ To know the technical details of how our formatter works and how to write test,
 
 ### Testing
 
-To run the tests, just run
+You can either use `cargo` or `just` to run tests. For simplicity and running tests real quick, use `cargo`.
+
+With `cargo`, you can run tests with using the `test` command:
+
+```shell
+# run tests
+cargo test
+
+# or use the shortcut
+cargo t
+```
+
+If you run `cargo t` from the root, it will run **all** tests of the whole repository. If you're inside a crate folder, `cargo` will run **tests of that crate**:
+
+```shell
+cd crates/biome_cli
+
+# it will run only the tests of the `biome_cli` crate
+cargo t
+```
+
+You can run **a single test** with cargo by passing the test name after the `test` command:
+
+```shell
+cd crate/biome_js_formatter
+
+cargo t quick_test
+```
```

Comment:
To avoid `cd` we could use `-p` option:

```sheel
cargo test -p biome_js_formatter -- quick_test
```

I am not sure if it is really equivalent.

Reply:
The purpose of the paragraph is to show/teach people that `cargo t` runs **only the tests of the crate** if you execute it from a folder crate, and not the root.